### PR TITLE
Website 2

### PR DESCRIPTION
**Pull request by Noviny**
Second pass on the website with much simpler gatsby

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# gatsby-starter-default
```
Comment (alexreardon) [inline website/README.md]: Do we need this file?

File: website/.browserslistrc [hunk 2]
```diff
@@ -1,8 +0,0 @@
-ie >= 11
```
Comment (alexreardon) [inline website/.browserslistrc]: is this still needed?

File: website/data/SiteConfig.js [hunk 3]
```diff
@@ -5,33 +5,6 @@ module.exports = {
   siteLogo: '/logos/logo-1024.png', // Logo used for SEO and manifest.
   siteUrl: 'https://react-beautiful-dnd.netlify.com', // Domain of your website without pathPrefix.
   pathPrefix: '/', // Prefixes all links. For cases when deployed to example.github.io/gatsby-advanced-starter/.
-  siteDescription: 'Beautiful, accessible drag and drop for lists with React.js', // Website description used for RSS feeds/meta description tag.
```
Comment (alexreardon) [inline website/data/SiteConfig.js]: amazing

File: website/data/SiteConfig.js [hunk 4]
```diff
@@ -5,33 +5,6 @@ module.exports = {
   siteLogo: '/logos/logo-1024.png', // Logo used for SEO and manifest.
```
Comment (alexreardon) [inline website/data/SiteConfig.js]: comment out?

File: website/gatsby-node.js [hunk 5]
```diff
@@ -1,156 +1,135 @@
 // @flow
-/* eslint-disable spaced-comment */
+/**
+ * Implement Gatsby's Node APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/node-apis/
+ */
 const path = require('path');
-const kebabCase = require('lodash.kebabcase');
+const startCase = require('lodash.startcase');
```
Comment (alexreardon) [inline website/gatsby-node.js]: can we use

Comment (alexreardon) [reply website/gatsby-node.js]: Capital for the first

File: website/gatsby-node.js [hunk 6]
```diff
@@ -1,156 +1,135 @@
...
       ).then((result) => {
         if (result.errors) {
           /* eslint-disable no-console */
           console.log(result.errors);
           reject(result.errors);
         }
 
-        const tagSet = new Set();
-        const categorySet = new Set();
+        console.log('DO THE THINGS', markdownPage);
```
Comment (alexreardon) [inline website/gatsby-node.js]: remove

File: website/package.json [hunk 7]
```diff
@@ -4,62 +4,20 @@
...
   },
-  "devDependencies": {
-    "cli-glob": "^0.1.0",
-    "gh-pages": "^1.0.0",
-    "remark-cli": "^4.0.0"
-  },
-  "keywords": [
-    "gatsby"
-  ],
+  "keywords": ["gatsby"],
```
Comment (alexreardon) [inline website/package.json]: update

File: website/package.json [hunk 8]
```diff
@@ -4,62 +4,20 @@
...
-    "lint:md": "remark content/posts/",
-    "write-good": "write-good $(glob 'content/posts/**/*.md')",
-    "format:js": "prettier '**/*.{js,jsx}' --write"
-  },
-  "remarkConfig": {
-    "plugins": [
-      "remark-preset-lint-recommended"
-    ]
+    "develop": "gatsby develop",
+    "format": "prettier --write 'src/**/*.js'"
```
Comment (alexreardon) [inline website/package.json]: remove as we are not using it in the rest of the project

File: website/src/layouts/index.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+import Helmet from 'react-helmet';
+/* eslint-disable-next-line import/extensions */
+import '@atlaskit/css-reset';
+
+import Header from '../components/Header';
+
+const TemplateWrapper = ({ children }: { children: () => Node }) => (
+  <div>
+    <Helmet
+      title="Gatsby Default Starter"
```
Comment (alexreardon) [inline website/src/layouts/index.js]: Use our page title?

File: website/src/layouts/index.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+      meta={[
+        { name: 'description', content: 'react-beautiful-dnd' },
```
Comment (alexreardon) [inline website/src/layouts/index.js]: update

File: website/src/layouts/index.js [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+    <Header />
+    <div
+      style={{
+        margin: '0 auto',
+        maxWidth: 960,
+        padding: '0px 1.0875rem 1.45rem',
+        paddingTop: 0,
+      }}
+    >
+      {children()}
```
Comment (alexreardon) [inline website/src/layouts/index.js]: update?

File: website/src/layouts/index.js [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+    </div>
+  </div>
+);
+
+TemplateWrapper.propTypes = {
+  children: PropTypes.func,
```
Comment (alexreardon) [inline website/src/layouts/index.js]: .isRequired? or flow?

File: website/.browserslistrc (hunk 2 above)
Comment (Noviny) [reply website/.browserslistrc]: N

File: website/gatsby-node.js [hunk 13]
```diff
@@ -69,18 +69,20 @@ exports.onCreateNode = ({ node, boundActionCreators, getNode }/* : NodeParams */
...
-      name = startCase(parsedFilePath.dir);
+      title = lowerCase(parsedFilePath.dir);
     }
     if (parsedFilePath.name !== 'index') {
       slug += `/${parsedFilePath.name.toLowerCase()}`;
-      name = startCase(parsedFilePath.name);
+      title = lowerCase(parsedFilePath.name);
     }
 
+    title = title.charAt(0).toUpperCase() + title.slice(1);
```
Comment (alexreardon) [inline website/gatsby-node.js]: probably not critical - but this will fail if the title is of length 1

Comment (alexreardon) [reply website/gatsby-node.js]: I think if we detect a title of length < 2 we should throw:

Comment (alexreardon) [reply website/gatsby-node.js]: if(title.length < 2) {

File: website/gatsby-node.js [hunk 14]
```diff
@@ -1,156 +1,135 @@
...
+    if (parsedFilePath.dir) {
+      slug += `/${parsedFilePath.dir.toLowerCase()}`;
+      title = lowerCase(parsedFilePath.dir);
     }
+    if (parsedFilePath.name !== 'index') {
+      slug += `/${parsedFilePath.name.toLowerCase()}`;
+      title = lowerCase(parsedFilePath.name);
+    }
+
+    title = title.charAt(0).toUpperCase() + title.slice(1);
```
Comment (alexreardon) [inline website/gatsby-node.js]: checking for short title or not worrying?

**Pull request merged**